project_timesheet_time_control/hooks.py

```diff
@@ -0,0 +1,12 @@
+# -*- coding: utf-8 -*-
+# © 2016 Pedro M. Baeza
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+
+def post_init_hook(cr, registry):
+    """Put the date with 00:00:00 as the date_time for the line."""
+    cr.execute("""
+        UPDATE account_analytic_line
+        SET date_time = to_timestamp(date || ' 00:00:00',
+                                     'YYYY/MM/DD HH24:MI:SS')
```

Comment:
Actually it's OK because `date_time` field is created by this addon, and AFAIK `post_init_hook` does not get executed on upgrades.